app/models/tasklist/sections/LTASection.scala

```diff
@@ -26,10 +26,18 @@ import play.api.mvc.Call
 case object LTASection extends Section {
 
   def removeAllUserAnswers(userAnswers: UserAnswers): UserAnswers =
-    userAnswers.removePath(JsPath \ "lta").get
+    userAnswers.removePath(JsPath \ "lta").get.removePath(JsPath \ "setup" \ "lta").get
 
   def removeAllUserAnswersAndNavigation(userAnswers: UserAnswers): UserAnswers =
-    userAnswers.removePath(JsPath \ "lta").get.remove(sectionNavigation).get
+    userAnswers
+      .removePath(JsPath \ "lta")
+      .get
+      .remove(sectionNavigation)
+      .get
+      .removePath(JsPath \ "setup" \ "lta")
```

Comment:
Requires a TODO to remove this upon triage implementation as will no longer be appropriate.